### PR TITLE
Fixed "issue" pluralization in write-release-notes.ps1

### PR DESCRIPTION
**Pull request by adamralph**
## Description

As [requested](https://github.com/microsoft/vstest-docs/pull/242#pullrequestreview-544238293).

### CODE REVIEW

File: scripts/write-release-notes.ps1 [hunk 1]
```diff
@@ -103,7 +103,7 @@ See the release notes [here](https://github.com/microsoft/vstest-docs/blob/maste
 
 ## $v
 
-### Issue Fixed
+### Issues Fixed
```
Comment (Haplois) [inline scripts/write-release-notes.ps1:106]:
```suggestion
### $(if ($issues.Length -eq 1) { 'Issue' } else { 'Issues' }) Fixed
```

**Review by Haplois — CHANGES_REQUESTED** [review]
Looks good, but let's handle the unlikely chance of only one issue is fixed on a release.

File: scripts/write-release-notes.ps1 (hunk 1 above)
Comment (adamralph) [reply scripts/write-release-notes.ps1:106]: I considered it and decided it was overkill, but you prefer, sure.

**Review by Haplois — APPROVED** [review]
LGTM. Thank you for your contribution.

**Pull request merged**